lte/modules/image_encode.py

```diff
@@ -0,0 +1,17 @@
+"""
+Encodes image from original format to JPEG, filtering out A channel if PNG
+"""
+
+import io
+
+from PIL import Image
+
+def encode(png_image):
```

Comment:
This function does not specify the parameter type or the type that it returns. Can you please add that.

Also, the image that is being passed in will be in the format of a numpy array in RGB format, not a PNG image.

Reply:
Added